CMakeLists.txt

```diff
@@ -73,45 +73,45 @@ if("${GYSELALIBXX_INCLUDE_TESTING_DEPENDENCIES}")
   if("${GYSELALIBXX_GTest_DEPENDENCY_POLICY}" STREQUAL "AUTO")
     find_package(GTest "1.12" QUIET)
     if(NOT "${GTest_FOUND}")
-      add_subdirectory("vendor/googletest/" "googletest") # SYSTEM)
+      add_subdirectory("vendor/googletest/" "googletest" SYSTEM)
     endif()
   elseif("${GYSELALIBXX_GTest_DEPENDENCY_POLICY}" STREQUAL "EMBEDDED")
-    add_subdirectory("vendor/googletest/" "googletest") # SYSTEM)
+    add_subdirectory("vendor/googletest/" "googletest" SYSTEM)
   elseif("${GYSELALIBXX_GTest_DEPENDENCY_POLICY}" STREQUAL "INSTALLED")
     find_package(GTest "1.12" REQUIRED)
   endif()
 endif()
 
-find_package(Kokkos 4.5.1 QUIET)
-if(NOT "${Kokkos_FOUND}")
-  ## Use Kokkos from `vendor/`
-  if("${Kokkos_ENABLE_CUDA}")
-    option(Kokkos_ENABLE_CUDA_CONSTEXPR "Whether to activate experimental relaxed constexpr functions" ON)
-    option(Kokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE "Whether to enable relocatable device code (RDC) for CUDA" ON)
-  endif()
-  add_subdirectory("vendor/kokkos/" "kokkos") # SYSTEM)
-endif()
+find_package(Kokkos 4.4.1 REQUIRED)
```

Comment:
I see 3 options:
- set the minimum required to 4.4.1
- remove Kokkos from the Spack 0.23 installation
- tweak the Spack 0.23 installation to be able to install Kokkos and Kokkos Kernels 4.5 or even 4.6.

@EmilyBourne Any opinion ?

Reply:
> set the minimum required to 4.4.1

Sounds ok to me. I'm not sure where this value comes from anyway

> remove Kokkos from the Spack 0.23 installation

Looking at #344 I realise that this is my least favourite solution as it requires us to build Kokkos which will significantly increase the build time. When I have to compile Kokkos I usually see ~20% of the compilation spent on Kokkos things.

> tweak the Spack 0.23 installation to be able to install Kokkos and Kokkos Kernels 4.5 or even 4.6.

I don't understand how this would help.

Reply:
> > remove Kokkos from the Spack 0.23 installation
> 
> Looking at #344 I realise that this is my least favourite solution as it requires us to build Kokkos which will significantly increase the build time. When I have to compile Kokkos I usually see ~20% of the compilation spent on Kokkos things.

See my answer in #344.

> > tweak the Spack 0.23 installation to be able to install Kokkos and Kokkos Kernels 4.5 or even 4.6.
> 
> I don't understand how this would help.

Because `find_package(Kokkos 4.5.1 QUIET)` in the CMakeLists.txt of gyselalibxx will not fail anymore.

Reply:
This line:
https://github.com/gyselax/gyselalibxx/blob/b2ad378f2de3425380a2256820bcb371fae9cc3d/CMakeLists.txt#L85
?

If we are looking for Kokkos 4.5.1 why are we installing Kokkos 4.4? Surely this is the issue?

Reply:
Because 4.5.1 is not available in Spack 0.23